### ISSUE TITLE
overfitting when training on cifar10

### ISSUE THREAD

**Issue by lowestbuaaer** [openai/improved-diffusion#91]
I trained a model on cifar10 with below settings.
`MODEL_FLAGS="--image_size 32 --num_channels 128 --num_res_blocks 3 --learn_sigma True --dropout 0.1"
DIFFUSION_FLAGS="--diffusion_steps 4000 --noise_schedule linear"
TRAIN_FLAGS="--lr 1e-4 --batch_size 16"`
but it overfits at 500k iterations.
optimal point locates at about 300k iterations.(sample 100 steps, FID 4.7)
Does anyone know possible reasons?

**Comment by 99-WSJ** [comment]
maybe you check the dropout== 0.3 in readme,

**Comment by yug125lk** [comment]
Hi, Did you solve it?
